valhalla/mjolnir/osmaccess.h

```diff
@@ -224,8 +240,11 @@ struct OSMAccess {
       uint16_t motorroad_tag : 1;
       uint16_t moped_tag : 1;
       uint16_t motorcycle_tag : 1;
-      uint16_t spare : 6;
+      uint16_t golf_cart_tag : 1;
+      uint16_t spare_ : 5;
     } fields;
+    // TODO: What exactly is 64 bits wide? I'm counting 16 + 32 = 48 bits?
+    uint16_t _spare;
     uint32_t v; // this should be 64bits wide for architectures who require word alignment
```

Comment:
I'm not quite sure what's going on here. I see a comment about something being 64 bits wide for alignment... but in this block of fields I'm only counting up 48. I'm occasionally both dense and bad at math though so maybe I missed something.

Reply:
its a bit complicated but let me explain. first, `AccessAttributes` is a `union` of a 16bit field and a 32bit field, that means the compiler will make the whole thing 32bits wide. so we shouldnt add the spare field, that wont do anything. the comment is really just saying that on 64bit platforms, `AccessAttributes` will get padded to 64bits because the other field that is in the outer `OSMAccess` struct is already 64bits (it used to be 32 and this one is 32 so they would both stop at 64).

Reply:
oooooooh! Thanks for the explanation; that makes sense.

Reply:
Reverted in the latest commit.